src/Grid/grid.jl

```diff
@@ -120,15 +120,16 @@ function addcellset!(grid::Grid, name::String, cellid::Union{Set{Int}, Vector{In
     grid
 end
 
-function addcellset!(grid::Grid, name::String, f::Function)
+function addcellset!(grid::Grid, name::String, f::Function; all = true)
     cells = Set{Int}()
     for (i, cell) in enumerate(getcells(grid))
-        all_true = true
+        pass = all
         for node_idx in cell.nodes
             node = grid.nodes[node_idx]
-            !f(node.x) && (all_true = false; break)
+            all && !f(node.x) && (pass = false; break)
```

Comment:
Perhaps store `f(node.x)` in a variable instead of calling it twice?

Reply:
Sure will do it tomorrow, thanks.

Reply:
But won't it be called only once per iteration? Because `all && ...` and `!all && ...` are like if statements.

Reply:
Actually storing `f(node.x)` is still useful to avoid calling it multiple times for different cells sharing the same node.

Reply:
Right, this should be free anyway so doesn't really matter.